docs/configuration.md

```diff
@@ -103,154 +103,194 @@ and print it out to the console.
 
 ```
 $ eventstored --what-if
-[  131, 1,13:05:59.721,INF]
-"ES VERSION:"             "22.10.0.0" ("tags/oss-v22.10.0"/"242b35056", "Thu, 10 Nov 2022 13:45:56 +0000")
-[  131, 1,13:05:59.732,INF] "OS ARCHITECTURE:"        X64
-[  131, 1,13:05:59.749,INF] "OS:"                     Linux ("Unix 4.19.128.0")
-[  131, 1,13:05:59.752,INF] "RUNTIME:"                ".NET 6.0.11/943474ca1" (64-bit)
-[  131, 1,13:05:59.752,INF] "GC:"                     "3 GENERATIONS" "IsServerGC: False" "Latency Mode: Interactive"
-[  131, 1,13:05:59.754,INF] "LOGS:"                   "/var/log/eventstore"
-[  131, 1,13:05:59.795,INF] MODIFIED OPTIONS:
-
-     CLUSTER SIZE:                            1 (Yaml)
-     RUN PROJECTIONS:                         None (Yaml)
-     WHAT IF:                                 true (Command Line)
+[11120, 1,11:28:40.519,INF] "OS ARCHITECTURE:"        X64 
```

Comment:
The output here should be of EventStore running on Linux with `--what-if` set since the example command is `eventstored --what-if`. Could you run this on Linux and replace the content with that output?

Reply:
Done